File: Special Day/Java/StudentGradingSystemTest.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.assertEquals;
import static org.junit.jupiter.api.Assertions.assertArrayEquals;

public class StudentGradingSystemTest{

    @Test
        public void testThatWeCollectInputFromUserAndStoreIt(){
        int [] [] storage = {
        {2, 2},
        {2 , 2}
        };
        int[][] actual = StudentGradingSystem.collectStudentScores(2, 2);
        int [][]expected = storage;
        assertArrayEquals(actual, expected);
        }

}

File: Special Day/Python/student_grading_system_test.py
from unittest import TestCase
import student_grading_system

class TestStudentGradingSystem(TestCase):

    def test_that_we_collect_input_from_user_and_store_it(self):
        storage = [[2,2],[2,2]]
        result =  collectStudentScores(2, 2)
        self.assertListEqual(result, storage)

